staging/operator-lifecycle-manager/test/e2e/catalog_exclusion_test.go

```diff
@@ -105,7 +105,7 @@ var _ = Describe("Global Catalog Exclusion", func() {
 		When("the operator group is annotated with olm.operatorframework.io/exclude-global-namespace-resolution=true", func() {
 			// Note: this test flakes on openshift CI but passes running on a local openshift cluster
 			// For some reason, the subscription does not transition to a failed state in time
-			It("[FLAKE] the broken subscription should resolve and have state AtLatest", func() {
+			It("the broken subscription should resolve and have state AtLatest", func() {
```

Comment:
Note: this was massaged into the original commit that downstreaming the global catalog exclusion work from upstream. This was added as a downstream-only change that I missed when originally opening the PR.